masschallenge/impact-api
# [AC-7983] P1: Office hours interface sends wrong end time when staff creates a block of sessions on behalf of mentor

## [AC-7983](https://masschallenge.atlassian.net/browse/AC-7983)

**Changes Introduced**
- Notification email to user uses the end time of the last session created on behalf of the user. 

**Testing**
- As a staff user, visit http://localhost:8000/api/v1/office_hour/
- Create a block of sessions on behalf of a mentor
- Confirm that the notification email to user uses the end time of the last session
